packages/docusaurus-plugin-content-blog/src/blogUtils.ts

```diff
@@ -92,6 +92,7 @@ export async function generateBlogFeed(
       link: normalizeUrl([siteUrl, permalink]),
       date,
       description,
+      content: post.content
```

Comment:
I don't believe this is ok.

The blog post content is MDX, and we must render it to a non-mdx form (HTML?) before passing the content to the feed.